include/parallelization/MPIHandler.hpp

```diff
@@ -0,0 +1,85 @@
+#include <mpi.h>
+#include <vector>
+#include "parser/GhostCell.hpp"
+
+
+#pragma once
+
+namespace E3D::Parallel {
+
+    /**
+     * @brief : Class to handle MPI communications between partitions
+     */
+    class MPIHandler {
+    public:
+
+        /**
+         *  @brief : ctor initializes MPI and populate rank,size
+         */
+        MPIHandler(int argc, char *argv[]);
```

Comment:
Maybe add the typical arguments in the description?

Reply:
Done